scripts/linux.sh

```diff
@@ -12,7 +12,7 @@ get_uuids() {
 }
 
 get_mountpoint() {
```

Comment:
Maybe this should be renamed to `get_mountpoints` to make it clearer that it _might_ return a (multi-line) list?